projects/driver_controls/test/test_control_stalk.c

```diff
@@ -0,0 +1,108 @@
+#include "control_stalk.h"
```

Comment:
Should probably outline that this is a functional test? Also a module comment on light to state mapping would be nice.

Reply:
The light output was more just for fun :P